scripts/release.js

```diff
@@ -79,7 +79,7 @@ const strategy = currentBranch !== 'develop' || forceBeta ? 'beta' : 'stable';
 function rollback(newVersion) {
   if (strategy === 'stable') {
     // reset master
-    shell.exec('git reset --hard origin master');
+    shell.exec('git reset --hard origin/master');
```

Comment:
what about doing `--soft` instead? this command is indeed the fix though

Reply:
It will leave the working files there, and I fail to see any value in that. Can you give your opinion?

Reply:
not sure either, just wondering if there could be any caution (if you e.g. released before pushing)?

Reply:
There is no rollback possible if the package is published. As far as I can see this is what we expect.